test-apps/display-test-app/package.json

```diff
@@ -83,12 +83,12 @@
     "eslint": "^6.8.0",
     "express": "^4.16.3",
     "fs-extra": "^8.1.0",
-    "internal-tools": "0.0.0",
+    "internal-tools": "workspace:*",
     "npm-run-all": "^4.1.5",
     "null-loader": "^0.1.1",
     "rimraf": "^3.0.2",
     "react": "^16.8.0",
-    "sass": "^1.29.0",
+    "sass": "~1.32.0",
```

Comment:
@SaemJeon is this in order to get rid of the warnings produced in ui-core?

Reply:
Yes. It was to remove division warnings.

Reply:
@bsteinbk / @DanEastBentley mind taking a look at the warnings that Saem is referring too? They happen when you update sass beyond 1.33, this can be reproduced locally on `master` by deleting the `common/config/rush/pnpm-lock.yaml`, running rush update and rush build.

Reply:
@calebmshafer / @SaemJeon -  How can I see the warnings you are talking about?

Reply:
@bsteinbk You can go to `update-rush` branch and change "sass": "~1.32.0" to "sass": "^1.29.0". Then, delete `common/config/rush/pnpm-lock.yaml` and run `rush update` and `rush build`. You should see the warnings.

Reply:
Pretty sure it could be done on master with the same changes.

Reply:
@calebmshafer it looks like it could take a day or two to address all the warnings - is that something that needs to be done right away? If so maybe @NancyMcCallB and I could pull this branch and play "whack-a-warning".

Reply:
Looks like there may be an automated way to update scss files....let me investigate, it may easier than I thought.

Reply:
Nope not right away.  In this PR @SaemJeon will just lock down to a prior version so you guys can get to it when you can.